apps/emqx_bridge_kinesis/src/emqx_bridge_kinesis.erl

```diff
@@ -30,8 +30,23 @@ roots() ->
 
 fields("config_producer") ->
     emqx_bridge_schema:common_bridge_fields() ++
-        emqx_resource_schema:fields("resource_opts") ++
-        fields(connector_config) ++ fields(producer);
+        fields("resource_opts") ++
+        fields(connector_config) ++
+        fields(producer);
+fields("resource_opts") ->
+    [
+        {resource_opts,
+            mk(
+                ref(?MODULE, "creation_opts"),
+                #{required => true, desc => ?DESC(emqx_resource_schema, "creation_opts")}
```

Comment:
This field was not required before.  Is the change intentional?

Reply:
Ops... it was not intentional... fixing it